extension/src/shared/reearth/types/reearth.ts

```diff
@@ -40,3 +41,11 @@ declare global {
     reearth?: Partial<ReEarth>;
   }
 }
+
+export type ReEarth = ReEarthV1 | ReEarthV2;
+
+export function isReEarthAPIv2(
+  reearth?: Partial<ReEarthV1> | Partial<ReEarthV2>,
+): reearth is Partial<ReEarthV2> {
+  return !!(reearth?.apiVersion && reearth?.apiVersion >= "2.0.0");
```

Comment:
Does `>= "2.0.0"` work? Because it's string.

Reply:
Emmm only work if we keep using x.y.z format and each one is a single number less than 10. I refactored this part with a version compare function.